src/content/glossary/index.md

```diff
@@ -534,6 +639,18 @@ The act of passing a [message](#message) from one account to another. If the des
 
 The third development stage of Ethereum, launched in October 2017.
 
+### mining {#mining}
+
+The process of verifying transactions and contract execution on the Ethereum blockchain in exchange for a reward in ether with the mining of every block.
+
+### mining pool{#mining-pool}
+
+The pooling of resources by miners who share their processing power and split [mining rewards](#mining-reward).
+
+### mining reward {#mining-reward}
+
+The amount of ether given to a miner that mined a new block.
+
```

Comment:
Think we should think post-merge here... more accurately this could be listed as "block reward"

```suggestion
### block reward {#block-reward}

The amount of ether rewarded to the producer of a new valid block.

```

Reply:
**Note:** This code suggestion actually would need to be relocated up to where it belongs in the list in ABC order